### PR TITLE
adds handholding

### PR DESCRIPTION
**Pull request by silicons**
target arms while pulling
image
image

### CODE REVIEW

**Comment by Hatterhat** [comment]
he fucking did it the absolute madman

**Comment by Magikarpet** [comment]
Ban this sick filth.

**Comment by kevinz000** [comment]
fuck you greytider

**Comment by AdamElTablawy** [comment]
8. Erotic/creepy stuff is not allowed.
No form of erotic roleplay is allowed on the servers, including things that could be construed as sexual by unwilling participants.

**Comment by deathride58** [comment]
Not even Citadel's gonna accept this degeneracy.

**Comment by Doppeltongue** [comment]
Disgusting filth on one of the cleanest times of the year.

**Comment by Okand37** [comment]
You can just emote it if you really want then passive grab the other person/pull them.

**Comment by MrStonedOne** [comment]
image

**Comment by AdamElTablawy** [comment]
Erotic/creepy stuff is not allowed.
No form of erotic roleplay is allowed on the servers, including things that could be construed as sexual by unwilling participants.Erotic/creepy stuff is not allowed.
No form of erotic roleplay is allowed on the servers, including things that could be construed as sexual by unwilling participants.Erotic/creepy stuff is not allowed.
No form of erotic roleplay is allowed on the servers, including things that could be construed as sexual by unwilling participants.Erotic/creepy stuff is not allowed.
No form of erotic roleplay is allowed on the servers, including things that could be construed as sexual by unwilling participants.

**Comment by F-OS** [comment]
@kevinz000 >adds handholding



YOU SICK FUCK

**Comment by Hatterhat** [comment]
image
image

**Comment by Slignerd** [comment]
otxfymgygq1vk041eo1

**Comment by QualityVan** [comment]
how dare you sully the sanctity of christmas like this

**Comment by F-OS** [comment]
@Slignerd delet this

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -362,7 +362,7 @@
 
 		add_logs(src, M, "grabbed", addition="passive grab")
 		if(!supress_message)
-			visible_message("<span class='warning'>[src] has grabbed [M] passively!</span>")
+			visible_message("<span class='warning'>[src] has grabbed [M] passively [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":""]!</span>")
```
Comment (F-OS) [inline code/modules/mob/mob.dm]: too lewd

**Comment by Hatterhat** [comment]
image
image
image
image
image

File: code/modules/mob/mob.dm [hunk 2]
```diff
@@ -362,7 +362,7 @@
 
 		add_logs(src, M, "grabbed", addition="passive grab")
 		if(!supress_message)
```
Comment (Xhuis) [inline code/modules/mob/mob.dm]: This isn't relevant, but I thought it might be funny if this was called `suplex_message` instead.

**Comment by F-OS** [comment]
> handholding
> **HANDHOLDING**

ERP REPORTED

File: code/modules/mob/mob.dm (hunk 2 above)
Comment (kevinz000) [reply code/modules/mob/mob.dm]: :joy:

**Comment by Hatterhat** [comment]
image

**Comment by MrStonedOne** [comment]
image

**Comment by Hatterhat** [comment]
image
@MrStonedOne HAS GONE COMPLETELY I N S A N E

**Comment by Suicidalpickles** [comment]
ANYTHING BUT HAND-HOLDING
BEGONE THOTS.

**Comment by AcuteCircle** [comment]
finally playable

**Comment by FantasticFwoosh** [comment]
Woah - Woah - **WOAH* im unwrapping my christmas present early of something to moan about for the remainder of this christmas season, thanks kev.

**Comment by duncathan** [comment]
fetish content is not allowed
I'd say "take it to citadel" but even their standards are above this

**Comment by FantasticFwoosh** [comment]
Extra tabling's for you this year.

**Comment by Slignerd** [comment]
Reopen this

**Comment by FantasticFwoosh** [comment]
Open your own server.

**Comment by outbools** [comment]
henlo reddit iksdee

**Comment by ACCount12** [comment]
Am I a pervert for unironically liking this PR?

**Comment by Daturix** [comment]
Add immediately!

**Comment by Hatterhat** [comment]
image
@ACCount12 yes
image
@Daturix y tho

**Comment by Kraseo** [comment]
*slams fist rhythmically* BRING :clap: IT :clap: BACK :clap:

**Comment by The-clown-known-as-Pepe** [comment]
My name is Commander Sherrie and this is my favorite PR on the tgstation.

**Comment by CPTANT** [comment]
Cutting off someones arm with an axe -> fine
holding someones hand -> EEEHWL CREEPY FETISH SHIT.

**Comment by megatiger78** [comment]
Handholding can only be acceptable if it can be utilised in combination with explosive holoparasites

**Comment by FantasticFwoosh** [comment]
Its not consensual if i suddenly change to grab intent in-between hand-holding, besides.

**Comment by Qbopper** [comment]
waiting for the inevitable lzimann 👎 on the OP

**Comment by Xhuis** [comment]
just saying, if you can't give a logical and factual reason this is fetish content, then closing it for such is silly

**Comment by Bawhoppen** [comment]
Are you people actually against this or is this just another meme

**Comment by MMMiracles** [comment]
people who call this fetish content probably bust a nut every time a girl bumps into them while passing by.

**Comment by Xhuis** [comment]
>implying they go outside

**Comment by Qustinnus** [comment]
anyone who romantically RPs online probably just wants to ERP and its even worse on this game because it results in metagaming as well.

can confirm because I was 12 once too

**Comment by Xhuis** [comment]
[citation needed]

**Comment by Qustinnus** [comment]
there are written theses regarding this subject that I'm willing to show as evidence.

**Comment by Qustinnus** [comment]
https://docs.google.com/document/d/1fVzDv6zX81-Np2CXojr8fTauDpG0X6xhTLe6ythx-7w/edit?usp=sharing @Xhuis your move catman

**Comment by Xhuis** [comment]
:think:

**Comment by qbmax32** [comment]
kev i am dissapoint

**Comment by Armhulen** [comment]
i will have to testmerge this one.

**Comment by FantasticFwoosh** [comment]
@Xhuis its the promotion of unsolicited touching/groping someones arm by holding onto it without consent. Also blame asian cartoons & manga for sexualising all of this stuff in the first place and being a reference point the people who keep PR'ing this stuff.

Its not innocent at all, you're all hardly squeaky clean reputation wise when you loudly broadcast your anime child avatars & post creepy artwork online, and it'll never be appropriately applied because it is **infantile** (like loli anime girls who didn't understand what i was referring to previously) and pointless to gameplay or human interaction.

There you got your logical reason, its sexual harrassment & used by emotionally immature members of our community.

**Comment by Qustinnus** [comment]
lol fwoosh go away you aren't a professional thesis writer like Dr. F. Ettuce Kontent

**Comment by KorPhaeron** [comment]
Why are people still seriously replying to fwoosh 3 months after he quit

**Comment by FantasticFwoosh** [comment]
Ok let me just gently drag you to the dorms for ERP or hold your hand uncomfortably for a long amount of time under the pretense that i am getting off on doing it. 

Clearer understood now? Besides the whole "**don't touch me, no sane living human being would let a stranger they don't know touch them**" issue its not a specific of what it is (the point you're arguing of "oh its just holding hands whats the big deal"), its how its used more importantly.

**Comment by Qustinnus** [comment]
@KorPhaeron yes that surely was a serious reply especially when I mentioned Dr. F. Ettuce Kontent

**Comment by hobbledehoy899** [comment]
roy wants more than just his hands held

**Comment by ColonicAcid** [comment]
@xhuis DO NOT INVOKE SWAGGY P'S MEME IN VAIN

**Comment by Rinoka** [comment]
i support this if two things are added:

1) when holding hands, if the other person responds with harm intent, they bitchslap you to the floor and you're left stunned and crying about being furever alone ;_;

2) using severed hands/arms in-hand makes you creepily hold them ala yoshikage kira

**Comment by CosmicScientist** [comment]
What happens if the target has no hands? What would the initiator hold?

**Comment by Bawhoppen** [comment]
They'd mime it

**Comment by PKPenguin321** [comment]
>unironically opened on christmas eve

😟 kev? everything ok?

**Comment by kyrahabattoir** [comment]
Respecting boundaries in ss13 XD

This Christmas keeps getting better and better, definitely a mighty fine vintage this year.

**Comment by ACCount12** [comment]
Why isn't this reopened yet?

**Comment by kevinz000** [comment]
fwoosh is a smoothbrain

File: code/modules/mob/mob.dm [hunk 3]
```diff
@@ -362,7 +362,7 @@
...
-			visible_message("<span class='warning'>[src] has grabbed [M] passively!</span>")
+			visible_message("<span class='warning'>[src] has grabbed [M][(zone_selected == "l_arm" || zone_selected == "r_arm")? " by their hands":" passively"]!</span>")
```
Comment (Hatterhat) [inline code/modules/mob/mob.dm]: this addition is a fucking crime

Comment (kevinz000) [reply code/modules/mob/mob.dm]: 1000 points gulag

Comment (CosmicScientist) [reply code/modules/mob/mob.dm]: Speaking of points, the bot didn't give you a feature tag - this would have been free!

**Comment by MrMaclang** [comment]
who forgot the [ feature ] tag

**Comment by ACCount12** [comment]
reopen or riot!

**Comment by Hatterhat** [comment]
feature more like ((((((feature))))))

**Comment by SgtxRootbeer** [comment]
reopen!

**Comment by Hatterhat** [comment]
keep it closed

**Comment by Qustinnus** [comment]
unexpected twist??

**Comment by Xhuis** [comment]
# *madman*

**Comment by Cyberboss** [comment]
# YOU HAVE BEEN LIED TO

https://github.com/tgstation/tgstation/commit/395ce00b

# _WAKE UP SHEEPLE_

**Comment by ACCount12** [comment]
REEEEEEEEEEEEEEEEEEEE

**Comment by The-clown-known-as-Pepe** [comment]
@optimumtact 

image

**Comment by Cyberboss** [comment]
# THE FRUIT SUPPRESSES TRUTH

untitled

**Comment by Cyberboss** [comment]
# 1 v 1 ME CITRUS

untitled

**Comment by The-clown-known-as-Pepe** [comment]
Inception

image

**Comment by The-clown-known-as-Pepe** [comment]
DEEPER

image

**Comment by Cyberboss** [comment]
I have won

**Comment by Hatterhat** [comment]
@optimumtact
>merging a PR
>a PR mainly supported by the cat discord
>a PR made by a coder in the cat discord who is a cat

smh

**Comment by Swagile** [comment]
first a admin blows up the entire server because cat ears and tails are removed

now this

the a b s o l u t e   s t a t e   o f   t g

**Comment by Cyberboss** [comment]
ITT people who don't understand git

**Comment by kyrahabattoir** [comment]
Shouldn't this be reopened? There is clearly popular interest.

**Comment by optimumtact** [comment]
that's not how this works.

the weak should fear the strong

**Comment by kyrahabattoir** [comment]
That's not how democracy works @optimumtact.

**Comment by FantasticFwoosh** [comment]
Ewww.

**Comment by Qustinnus** [comment]
@kyrahabattoir since when is this a democray

**Comment by Hatterhat** [comment]
nice

**Comment by SumWhan** [comment]
Did someone say they needed a dose of freedom?
democracy

**Comment by adamkad1** [comment]
citadel actually took it

**Comment by adamkad1** [comment]
> first a admin blows up the entire server because cat ears and tails are removed
> 
> now this
> 
> the a b s o l u t e s t a t e o f t g

i mean, the pseudoremoval of felinids was pointless and people who voted to remove them are pointless
I dont play them though, im just throwing my logic applied opinion. Literally nothing was achieved by that

**Pull request merged**